nx-X11/programs/Xserver/hw/nxagent/Window.c

```diff
@@ -1476,16 +1473,16 @@ void nxagentConfigureWindow(WindowPtr pWin, unsigned int mask)
           }
         }
         fprintf(stderr, "\n");
-
-        if (children_return)
-        {
-          XFree(children_return);
-        }
       }
```

Comment:
I think it is okay to let xfree here directly after the while loop.

Reply:
Yes, but the proposed version is also correct and falls in line with the other changes.